src/main.rs

```diff
@@ -383,7 +407,13 @@ fn main() {
     .map(|()| log::set_max_level(log_level))
     .expect("Expected to be able to setup logger");
 
-    start_vmm(cmd_arguments);
+    if let Some(backend_command) = cmd_arguments.value_of("net-backend") {
+        start_net_backend(backend_command);
+    } else if let Some(backend_command) = cmd_arguments.value_of("block-backend") {
+        start_net_backend(backend_command);
+    } else {
+        start_vmm(cmd_arguments);
+    }
```

Comment:
Adding some validation here could be nice. I'm thinking about checking that we don't get `--net-backend` and `--block-backend` together. Also, almost all other command arguments are incompatible with both `--net-backend` and `--block-backend`, except the logging ones.

Reply:
Done!